tests/methods/avas-8/input.dat

```diff
@@ -0,0 +1,114 @@
+# Test the pi orbital selection of AOSubspace on overlapped planes C20 molecule
+
```

Comment:
Add in the description that this uses forte to find the planes of a fullerene molecule in forte

Reply:
The planes are figured by the user, not Forte.